routers/web/user/profile.go

```diff
@@ -263,6 +263,7 @@ func Profile(ctx *context.Context) {
 			TopicOnly:          topicOnly,
 			Language:           language,
 			IncludeDescription: setting.UI.SearchRepoDescription,
+			Pinned:             util.OptionalBoolFalse,
```

Comment:
I am still not sure if we should store this on the repo level.
I thought you wanted to implement a GitHub-like approach were the pinned repositories "float" above the repo list. With that approach, you then don't need to store this on the repo level.

Reply:
Reverted